.gitignore

```diff
@@ -15,4 +15,6 @@ working*
 
 App.config.mine
 src/packages/*
-!src/packages/repositories.config
\ No newline at end of file
+!src/packages/repositories.config
```

Comment:
I think this is just GitHub messing up the comparison. That line didn't change at all, the pluses and minuses are showing what GitHub thinks was added and subtracted.